middleware/routes/claims.js

```diff
@@ -57,4 +58,17 @@ router.post('/:vin', authMiddleware, async (req, res) => {
   }
 });
 
+// Add images to a claim using the Detailed Claim view.
+router.post('/:vin/:files', authMiddleware, async (req, res) => {
+  try {
```

Comment:
Remove `try/catches`.

See here: https://expressjs.com/en/guide/error-handling.html

Reply:
Do this later.